tvbwidgets/ui/spacetime_widget.py

```diff
@@ -169,48 +174,63 @@ def _create_matplotlib_graphs(self):
         plt.close(self.fig)      
 
     def _add_options(self):
-        self.options = HBox(layout = Layout(width = '500px'))
+        self.options = HBox(layout = Layout(width = '800px'))
+        conduction_speed = self.connectivity.speed
+        max_time = self.connectivity.tract_lengths.max() / conduction_speed
+        min_time = self.connectivity.tract_lengths.min() / conduction_speed
         self.option_conduction_speed = FloatText(
-                value=1.0,
+                value=conduction_speed,
```

Comment:
Sorry, I know I said take this from the connectivity, but I think this should actually be 1 when first rendering the plots 😅 please change it back